packages/apify/src/configuration.ts

```diff
@@ -18,6 +18,9 @@ export interface ConfigurationOptions extends CoreConfigurationOptions {
     proxyStatusUrl?: string;
     isAtHome?: boolean;
     userId?: string;
+    inputSecretsPrivateKeyPassphrase?: string;
+    inputSecretsPrivateKeyFile?: string;
+    inputKey?: string;
```

Comment:
not sure if it makes sense, but this option is apparently not mapped from env vars

Reply:
It is probably not necessary here as the key is in crawlee core CoreConfigurationOptions.